static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,144 @@
+# Shared Storage on NFS
+
+In the modern software development environment, teams are working together on
+same dataset to get the results. It became necessary that data is accessible and
+every team member has a same updated dataset. For this example, we will be using
+NFS (Network File System) for storing and sharing files on the network. This
+allows you to have better resource utilization such as ability to store large
+disk consuming dataset on a single host machine.
```

Comment:
just large datasets

i'm not sure I understand this sentence, what exactly the single host machine does here?

Reply:
@shcheklein we are storing the dataset on a single machine and people are accessing it from there local machines. Not everyone wants to copy the dataset locally on their machine and work.